types/karma/index.d.ts

```diff
@@ -242,17 +318,77 @@ export interface ConfigOptions {
      * you can specify how many browsers should be running at once at any given point in time.
      */
     concurrency?: number;
+    /**
+     * When true, this will append the crossorigin attribute to generated script tags,
+     * which enables better error reporting for JavaScript files served from a different origin
+     * @default true
+     */
+    crossOriginAttribute?: boolean;
+    /**
+     * If null (default), uses karma's own context.html file.
+     * @default null
+     */
+    customContextFile?: null | string;
```

Comment:
while karma does support `null` here (as it does a lazy falsy check) and below, IMO we should stick with the following for consistency:

`customContextFile?: string;`

same with the ones below.

Reply:
@43081j agree, redone that part via ae7e43ddafd7bab51d0dfd41638d2ea6792353e8